pages/index.js

```diff
@@ -22,27 +22,28 @@ see the source: https://github.com/hackclub/hackfoundation.org
       <HeadObject />
       <main className="m-10 md:mx-12 lg:mx-24 md:my-2">
         <section className="flex flex-col md:grid md:grid-cols-3">
-            <section className="md:mr-16 md:col-span-2 flex flex-col space-y-12 text-lg lg:text-2xl">
-              <p><b>The Hack Foundation</b> was founded in 2016 to support the operations of Hack Club, a movement of 13,000 teenage programmers learning and making together.</p>
-              <p>Today, the foundation partners with over 330 non-profit organizations including Hack Clubs, hackathons, Little League teams, newspapers, and everything in between to act as their legal and financial entity.</p>
-              <p>Our innovative program allows anyone with a vision to build something meaningful because we believe in a world where social entrepreneurs who are are working toward better and stronger communities face no barriers towards doing good.</p>
+            <section id="main" className="md:mr-16 md:col-span-2 flex flex-col space-y-12 text-lg lg:text-xl">
+              <p>
+                <b>The Hack Foundation</b> was founded in 2016 to serve as the legal entity for <a href="https://hackclub.com" target="_blank">Hack Club</a>, a nonprofit network of high schoolers creating the coding education they wish they could have.
+              </p>
+              <p>
+                Today, The Hack Foundation is the largest <a href="https://en.wikipedia.org/wiki/Fiscal_sponsorship">fiscal sponsor</a> for teen-led projects in the world.</p>
+              <p>
+                Hundreds of diverse groups with annual budgets from $100 to $250,000 are powered by The Hack Foundation. These include Hack Clubs, hackathons, Little League teams, community organizations, and <a href="https://bank.hackclub.com/hq/">Hack Club HQ itself</a>. Partnered organizations receive the benefits of 501(c)(3) nonprofit status, administrative and backoffice support, and financial oversight.
+              </p>
+              <p>
+                The Hack Foundation was founded by a 16 year old. The organization and all its software has been built over the years by teenagers at Hack Club. Today's supporters include...
+              </p>
+              <p>
```

Comment:
Why is there an extra tag pair here?

Reply:
Very likely my bad. Please do an opinionated pass on it and fix any issues I introduced